stdlib/2.7/__builtin__.pyi

```diff
@@ -529,7 +529,7 @@ class dict(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     @overload
     def __init__(self, map: Mapping[_KT, _VT], **kwargs: _VT) -> None: ...
     @overload
-    def __init__(self, iterable: Iterable[Tuple[_KT, _VT]], **kwargs: _VT) -> None: ...
+    def __init__(self, iterable: Iterable[Union[Tuple[_KT, _VT], Sequence[_VT]]], **kwargs: _VT) -> None: ...
```

Comment:
Hmm, I'd forgotten about that.

(Following up on PR #257's main comment thread, since I found myself writing a long comment and the main comment threads feel like a better medium for that.)